tests/eval_files/trio105_anyio.py

```diff
@@ -0,0 +1,13 @@
+# NOTRIO
+import anyio
+
+
+async def foo():
+    nursery = anyio.open_nursery()
```

Comment:
Probably worth a comment that all this is fine because anyio calls the equivalent construct a "taskgroup", following asyncio?

Reply:
Following curio, actually. Asyncio got task groups much later than either curio or AnyIO.